### DISCUSSION TITLE
How do I create new notes from template?

### DISCUSSION THREAD

**Discussion by vmcjoao** [SilentVoid13/Templater#1425] [Help]
So my problem here is that I have a daily note template and it's header looks like this:

`<< [[{{yesterday}}|Yesterday]] || [[<% tp.date.now("YYYY-MM", 1) %>]] || [[{{tomorrow}}|Tomorrow]] >>`

This middle part links to my monthly note, and if there isn't one yet it creates it with the name of the current month, but I coudn't manage to write a command that used my monthly note template when creating it, so rn I'm creating the month note first to then be able to use this link in the daily notes. Is there a way to make this work? I also don't know how to select the folder that the new note will go

**Comment by Zachatoo** [comment]
I answer this question frequently, I've written up an answer [here](https://zachyoung.dev/posts/how-to-fix-links-in-daily-notes-not-using-daily-notes-template).

In short, you need to include the full path to the file in the link to determine where the file gets created, and use folder templates to ensure newly created files in that folder use your desired template.
